feg/gateway/docker/go/Dockerfile

```diff
@@ -67,6 +67,7 @@ RUN printenv > /etc/environment
 
 COPY feg/radius/lib/go/ /radius/lib/go
 COPY feg/radius/src/go.* /radius/src/
+COPY orc8r/lib/go/go.* /orc8r/lib/go/
```

Comment:
Do the relative paths actually match up here, with radius being placed at `/radius/src/` instead of `/feg/radius/src/`?